pkg/driver/node/mounter/systemd_mounter.go

```diff
@@ -49,7 +49,7 @@ func (m *SystemdMounter) IsMountPoint(target string) (bool, error) {
 		return false, err
 	}
 
-	mountPoints, err := m.MountLister.ListMounts()
+	mountPoints, err := m.Mounter.List()
```

Comment:
Can we add a comment here then that we're using `Mounter.List` instead of `Mounter.IsMountPoint` because we also want to verify that the mount point is an `mount-s3` mount point as well as being a mount point